pyvista/plotting/plotting.py

```diff
@@ -1082,10 +1092,15 @@ def add_mesh(self, mesh, color=None, style=None, scalars=None,
             mesh.
 
         render_points_as_spheres : bool, optional
+            Render points as spheres rather than dots.
 
         render_lines_as_tubes : bool, optional
+            Show lines as thick tubes rather than flat lines.  Control
+            the width with ``line_width``.
 
         smooth_shading : bool, optional
+            Enable smooth shading when ``True``.  Computes point
+            normals enables VTK's phong shading algorithm.
```

Comment:
I can't parse the second sentence and I'm not sure what the correct version is.

Reply:
```suggestion
            Enable smooth shading when ``True`` using either the 
            Gouraud or Phong shading algorithm.  When ``False``, use
            flat shading.
```

Reply:
Good point about the version.  VTK 8.2 and new uses Phong and 9 used Gouraud.

Reply:
You give me too much credit. I just meant that I could think of at least two ways to fix that sentence :smile: